main2.m

```diff
@@ -119,32 +130,35 @@
     %     %save meaninful data, estimated data, meaninful data no offset
     save(strcat('data/',experimentName,'/dataset.mat'),'dataset')
 end
+
 %% Data exploration
 % Plot ftDataNoOffset and/vs estimatedFtData
-for i=4:4
-    %     for i=1:size(input.ftNames,1)
-    FTplots(struct(input.ftNames{i},dataset.ftData.(input.ftNames{i}),strcat('estimated',input.ftNames{i}),dataset.estimatedFtData.(input.ftNames{i})),dataset.time);
-end
+if( scriptOptions.printPlots )
```

Comment:
I see the changes in main2. I will replicate them in main.m which is a restructured version of main2.m and at least from my point of view is a bit cleaner. It also has most the plots grouped so also enabling or disabling pots its "easier" , it also includes some options in the plotting depending if you are intrested in having the comparison, or just the wrench space graphs as well as without the offset or with the offset.